dotCMS/src/main/java/com/dotmarketing/portlets/director/action/DirectorAction.java

```diff
@@ -500,7 +500,7 @@ else if (workingLink.isLocked()) {
 	
 	                if (InodeUtils.isSet(identifier.getInode()) && InodeUtils.isSet(htmlPageIdentifier.getInode()) && InodeUtils.isSet(containerIdentifier.getInode())) {
 	                    MultiTree mTree = new MultiTree(htmlPageIdentifier.getInode(),containerIdentifier.getInode(),identifier.getInode());
-	                    java.util.List<MultiTree> treeList=  MultiTreeFactory.getMultiTree(htmlPage, container);
+	                    java.util.List<MultiTree> treeList=  MultiTreeFactory.getMultiTrees(htmlPage.getIdentifier(), container.getIdentifier());
 	                    for (int i = 0; i < treeList.size(); i++) {
 	                    	if(treeList.get(i).getChild().equals(identifier.getInode())){
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "getInode"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)